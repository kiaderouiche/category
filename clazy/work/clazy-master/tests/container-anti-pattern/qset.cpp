#include <QtCore/QSet>

void test()
{
    QSet<int> s1, s2;
    s1.intersects(s2); // OK
    s1.intersect(s2).isEmpty(); // Warning
}
