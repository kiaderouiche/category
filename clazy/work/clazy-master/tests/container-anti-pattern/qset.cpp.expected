container-anti-pattern/qset.cpp:7:5: warning: Use QSet::intersects() instead [-Wclazy-container-anti-pattern]
